app/javascript/pages/admin/Events/index.js

```diff
@@ -34,27 +34,28 @@ const columns = deleteEvent => [
   {
     Header: 'Title',
     accessor: 'title',
+    sortable: true,
   },
   {
     Header: 'Office',
     accessor: 'office.name',
-    sortable: false,
+    sortable: true,
   },
   {
     Header: 'Description',
     accessor: 'description',
-    sortable: false,
+    sortable: true,
   },
   {
     Header: 'Start',
     accessor: 'startsAt',
-    sortable: false,
+    sortable: true,
```

Comment:
Is ReactTable smart enough to recognize this column to be date & time and apply sorting accordingly, instead of sorting alphabetically?

Reply:
Good point I will go check

Reply:
Looks like it:

<img width="935" alt="screen shot 2018-09-12 at 4 52 05 pm" src="https://user-images.githubusercontent.com/2517811/45407266-3be08480-b6ac-11e8-8a40-17a9da676835.png">